packages/connect/CHANGELOG.md

```diff
@@ -1,15 +1,66 @@
-|             Package              | Stable | Canary |
-| :------------------------------: | :----: | :----: |
-|       npm @trezor/connect        | 9.3.0  |   -    |
-|     npm @trezor/connect-web      | 9.3.0  |   -    |
-| npm @trezor/connect-webextension | 9.3.0  |   -    |
+|             Package              | Stable |    Canary    |
+| :------------------------------: | :----: | :----------: |
+|       npm @trezor/connect        | 9.3.0  | 9.3.1-beta.1 |
+|     npm @trezor/connect-web      | 9.3.0  | 9.3.1-beta.1 |
+| npm @trezor/connect-webextension | 9.3.0  | 9.3.1-beta.1 |
 
-|     Deployment     | Stable | Canary |
-| :----------------: | :----: | :----: |
-| connect.trezor.io/ | 9.3.0  |   -    |
+|     Deployment     | Stable |    Canary    |
+| :----------------: | :----: | :----------: |
+| connect.trezor.io/ | 9.3.0  | 9.3.1-beta.1 |
 
 Use the persistent link [connect.trezor.io/9](https://connect.trezor.io/9/) to access the latest stable version of Connect Explorer.
 
+# 9.3.1-beta.1
+
+Remove support for legacy passprahse support (7bb57cf)
+AuthenticateDevice verify CA cert extensions (4ba4445)
+Add support for T3B1 (f7f1705)
+
+    - chore(connect): Add Ambire Browser Extension to knownHosts (cbe81ab)
+    - chore(connect): remove ethereum-cryptography (18851d1)
+    - chore(connect): split public and internal settings type (3aab4ea)
+    - chore(connect): remove useCoreInPopup from explorer and tests (fee60c1)
+    - chore(connect): add new CA pubkeys and update timestamp (48ec951)
+    - chore(connect): refactoring of the CheckFirmwareAuthenticity to support local binary (and wont download it from web, when its available in the suite) (82ab513)
+    - docs(connect): describe coreMode (a5238d6)
+    - fix(connect): disallow interaction-less fw update when switching fw type (4936939)
+    - fix(connect): validate x509 asn1 component (bc13fe2)
+    - fix(connect): handle missing language binary (954dbb9)
+    - fix(connect): proceed with fw install even when language data fails to download (b40d1b7)
+    - test(connect): x509 extensions parser (163c82c)
+    - test(connect): update to min version fw 1.0.9 and 2.3.0 (544c311)
+
+## connect-ui
+
+    - remove bridge references, deeplink to suite desktop (98e98a4)
+
+## connect-plugin-stellar
+
+    - update stellar sdk (89cf20e)
+    - update trends libs (70b9c11)
```

Comment:
we probably want to release these as well. these are separate packages.

Reply:
Yes, we need to release those manually, right?

Reply:
yes. I think I can do it, I will check